csrc/velox/functions/text/vocab_ops.h

```diff
@@ -0,0 +1,26 @@
+
+#pragma once
+
+#include <vector>
+#include "velox/functions/Udf.h"
+#include "velox/type/Type.h"
+#include "vocab.h"
+
+namespace facebook::torcharrow::functions {
+template <typename T> 
+struct lookup_indices {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool
+  call(velox::exec::ArrayWriter<int64_t> &result,
+       const arg_type<std::shared_ptr<Vocab>> &vocab,
+       const arg_type<velox::Array<velox::Varchar>> &tokens) {
+    std::vector<std::string> input;
+    for(const auto& token:tokens){
```

Comment:
Could you also try explicitly typing `token` instead of using `auto`? I.e. `for(const velox::Varchar& token:tokens){`?

Reply:
I think I did try that as well, but didn't work.